app/controllers/attachments_controller.rb

```diff
@@ -4,14 +4,12 @@
 class AttachmentsController < ApplicationController
   include FragmentCachable
 
-  before_action :authenticate_attachment
   around_action :cache_attachments
+  before_action :authenticate_attachment
```

Comment:
This ought to have caused a failing spec, but the previous commit passes. Let's add a test for this.

Reply:
Not clear what test needs to be added...

Reply:
Ah I remember now... it an integration spec which is failing without this change:
  `rspec ./spec/integration/errors_spec.rb:121`

I will amend the commit message to make this clearer when rebasing.